misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -85,38 +85,14 @@ class MiskCommonServiceModule : KAbstractModule() {
 
   @Provides
   @Singleton
-  fun provideServiceManager(
+  internal fun provideServiceManager(
     injector: Injector,
     services: List<Service>,
-    listeners: List<ServiceManager.Listener>
-  ): ServiceManager {
-    // NB(mmihic): We get the binding for the Set<Service> because this uses a multibinder,
-    // which allows us to retrieve the bindings for the elements
-    val serviceListBinding = injector.getBinding(serviceSetKey)
-    val invalidServices = serviceListBinding
-        .acceptTargetVisitor(CheckServicesVisitor())
```

Comment:
We still have this check somewhere else?

Reply:
Line 99 in the updated MiskServiceModule. The diff is not very smart because it was duplicate code that I removed.